envs/parentv2.1.yml

```diff
@@ -0,0 +1,167 @@
+name: parentv2.1
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - aom=3.5.0=hf0c8a7f_0
```

Comment:
How did you create this file? You probably don't need most of these specified here. What I mean by this is, sometimes you install "numpy" and it install bunch of dependencies. In conda, it's sufficient to specify just `numpy` and let conda figure out the dependencies when it's installing things. Here, you seem to have specified both core packages (like numpy) and its dependencies, which may cause issues across OSs.

Another thing is `blosc=1.21.5=heccf04b_0`, in a line like this `blosc` specifies the tool, `1.21.5` specifies the version number and `heccf04b_0` specifies an OS-specific hash. So you probably don't need to add the last bit, if you want this to run on a Linux machine etc.

Reply:
I used the command `conda env export > mynewenv.yml`. Is there another way to generate the yml file?

Reply:
There are two ways to create this file:
* Manually (as we discussed on Slack)
* You can try this `conda env export --from-history > my_env.yml` This should only export the top level dependency tools and shouldn't include anything else. But sometimes doesn't work, so give it a whirl!

If that doesn't work, we can maybe look into doing it manually.

Reply:
The --from-history command worked. Thank you!